src/components/tools/Partials/About.svelte

```diff
@@ -29,85 +29,79 @@
   });
 </script>
 
-<style>
-</style>
-
-<div class="bx--grid">
-  <!-- About -->
-  <div class="bx--row">
-    <div class="bx--col-lg-12">
-      <h2>About the Tool</h2>
-      <slot name="description">
-        <em>[Provide content describing the tool]</em>
-      </slot>
-    </div>
+<!-- About -->
+<div class="bx--row">
+  <div class="bx--col-lg-12">
+    <h2>About the Tool</h2>
+    <slot name="description">
+      <em>[Provide content describing the tool]</em>
+    </slot>
   </div>
-  <!-- Datasets -->
-  <div class="bx--row">
-    <div class="bx--col-lg-12">
-      <h3>Data Sources</h3>
-      <slot name="sources">
-        <p>
-          The following list of datasets were used to create this tool. Download
-          data visualized in the charts by clicking the Download Chart button.
-          For more download options follow the links below.
-        </p>
-        {#if !datasets || !datasetList}
-          <em>[Provide a list of datasets used for the tool]</em>
-        {:else}
-          {#await datasetList}
-            <InlineLoading description="Loading list of datasets..." />
-          {:then items}
-            {#each items as item}
-              <div class="bx--row source">
-                <div class="bx--col-lg-2">
-                  <img
-                    src="/img/logos/{item.logo}"
-                    class="source-logo"
-                    alt="data provider logo"
-                  />
-                </div>
-                <div class="bx--col-lg-12">
-                  <h4>{item.title}</h4>
-                  <h5>{item.publisher}</h5>
-                  <p class="source-text">{item.description}</p>
-                  <p class="source-text">Download dataset:</p>
-                  <ul class="source-list">
+</div>
+<!-- Datasets -->
+<div class="bx--row">
+  <div class="bx--col-lg-12">
+    <h2>Data Sources</h2>
+    <slot name="sources">
+      <p>
+        The following list of datasets were used to create this tool. Download
+        data visualized in the charts by clicking the Download Chart button. For
+        more download options follow the links below.
+      </p>
+      {#if !datasets || !datasetList}
+        <em>[Provide a list of datasets used for the tool]</em>
+      {:else}
+        {#await datasetList}
+          <InlineLoading description="Loading list of datasets..." />
+        {:then items}
+          {#each items as item}
+            <div class="bx--row source">
+              <div class="bx--col-lg-2">
```

Comment:
When the viewport width narrows to 1056px and the grid switches from "large" to "medium"  (see [this list of breakpoints for the Carbon Grid](https://www.carbondesignsystem.com/guidelines/2x-grid/implementation#responsive-options_)) these two columns get stacked and the result is that the logo image becomes huge and blurry. We could fix this either by 1. not displaying the image at smaller screen widths, or 2. adding CSS classes with `bx--col-md-` to shift the layout (e.g. 1 column for the image and 7 columns for the text (there are 8 columns in the medium size grid)).

Reply:
Maybe for now we can just add the medium column css classes since we already agreed we will be coming back to fixing the layout for mobile devices later?

Reply:
Sounds good. I'll make this update.